docs/docs.less

```diff
@@ -248,35 +288,78 @@ dl:not(.hxList) {
     }
   }
 }
+//end:EXAMPLE component
 
-iframe[src*="codepen.io"] {
-  border: 2px solid @gray-900;
-  height: 25rem;
-  width: 100%;
+// "media" object (mainly for listing icons in Icons docs)
+.media {
+  display: flex;
+  padding: 1rem;
+
+  &__icon {
+    flex-shrink: 0;
+  }
+
+  &__body {
+    flex-grow: 1;
+    margin-left: 1rem;
+  }
 }
 
-hx-icon {
-  &.native-support {
-    color: @green-500;
+// element metadata container
+.metadata {
+  #Box.md();
+  background-color: @cyan-50;
+  border: 1px solid @cyan-100;
+  box-shadow: inset 0.5rem 0 0 0 @cyan-100;
+
+  dt {
+    #grid.span(3);
   }
 
-  &.polyfill-needed {
-    color: @red-900;
+  dd {
+    #grid.span(9);
   }
 }
 
-dl.memberList {
-  max-width: 80ch;
+/* ---------------------------------------- *\
+ * PAGE CLASSES
+\* ---------------------------------------- */
+
+/* ----- Element Page Styling ----- */
+.element-content {
+  // Attributes, Methods, and Properties
+  dl.memberList {
+    max-width: 80ch;
+  }
 }
 
-/* Beta Auto Grid Styles */
-.hxRow.hxAuto > .hxCol { flex-grow: 0; }
-.hxRow .hxCol.hxAuto { flex: 0 0 auto; }
+/* ----- Front Page Styling ----- */
+.front-page {
+  background-image: linear-gradient(0deg, @cyan-100, @gray-0 20%);
+}
+
+/* ----- Guide Page Styline ----- */
+.guide-content {
+  // polyfills compatibility table
+  hx-icon {
+    &.native-support {
+      color: @green-500;
+    }
+
+    &.polyfill-needed {
+      color: @red-900;
+    }
+  }
+}
 
 /* ----- Test Page Styling ----- */
 .test-content {
+  box-shadow: inset 0 0 0 0.5rem @yellow-900;
+  overflow: visible !important; // allow banner to be sticky
+
   section {
     > * {
+      // FIXME: breaks positioned elements
```

Comment:
- [ ] FIXME: Adds top margin to positioned elements (shifting them down, too far)